server/sonar-process-monitor/src/main/java/org/sonar/application/process/ProcessLauncherImpl.java

```diff
@@ -163,6 +165,22 @@ private static void writeJvmOptions(EsCommand esCommand) {
     }
   }
 
+  private static void writeElasticsearchSettings(EsCommand esCommand) {
+    Path elasticsearchSettingsFile = esCommand.getElasticsearchSettingsFile();
+    String elasticsearchSettings = esCommand.getElasticsearchSettings()
+      .entrySet()
+      .stream()
+      //FIXME yaml formatting and escaping
+      .map(e -> e.getKey() + ": " + e.getValue())
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)